test/sqllogictest/transform/union.slt

```diff
@@ -0,0 +1,88 @@
+# Copyright Materialize, Inc. All rights reserved.
```

Comment:
With INTERSECT, I get the following plan:

```
materialize=> EXPLAIN SELECT * FROM t2  EXCEPT ALL SELECT * FROM t1  INTERSECT ALL SELECT * FROM t1;
          Optimized Plan           
-----------------------------------
 %0 =                             +
 | Get materialize.public.t2 (u11)+
                                  +
 %1 =                             +
 | Get materialize.public.t1 (u7) +
 | Negate                         +
                                  +
 %2 =                             +
 | Get materialize.public.t1 (u7) +
                                  +
 %3 =                             +
 | Get materialize.public.t1 (u7) +
 | Negate                         +
                                  +
 %4 =                             +
 | Union %2 %3                    +
 | Threshold| Negate              +
 | Negate                         +
                                  +
 %5 =                             +
 | Union %0 %1 %4                 +
 | Threshold
(1 row)
```
The result is correct, I just wonder about %4, where ```Negate``` is mentioned twice within the same operator.